openapi.yaml

```diff
@@ -0,0 +1,404 @@
+swagger: "2.0"
+info:
+  description: "reader api service"
+  version: "0.0.1"
+  title: "Reader API"
```

Comment:
1. should follow this order:
- title
- description
- version
2. description: should provide more details "this api service served to end users or third-party apps/services"
3. version: this api specs is for version 1.0.0